src/families/solana/validator-app/index.ts

```diff
@@ -0,0 +1,105 @@
+import { Cluster } from "@solana/web3.js";
+import { AxiosRequestConfig, AxiosResponse } from "axios";
+import { compact } from "lodash/fp";
+import network from "../../../network";
+
+export type ValidatorsAppValidatorRaw = {
+  active_stake?: number | null;
+  commission?: number | null;
+  total_score?: number | null;
+  vote_account?: string | null;
+  name?: string | null;
+  avatar_url?: string | null;
+  delinquent?: boolean | null;
+  www_url?: string | null;
+
+  // not used data
+  /*
+  network: string;
+  account: string;
+  keybase_id: string | null;
+  www_url: string | null;
+  details: string | null;
+  created_at: Date;
+  updated_at: Date;
+  admin_warning: string | null;
+  root_distance_score: number;
+  vote_distance_score: number;
+  skipped_slot_score: number;
+  software_version: string;
+  software_version_score: number;
+  stake_concentration_score: number;
+  data_center_concentration_score: number;
+  published_information_score: number;
+  security_report_score: number;
+  data_center_key: string;
+  data_center_host: string | null;
+  authorized_withdrawer_score: number;
+  autonomous_system_number: number;
+  latitude: string;
+  longitude: string;
+  url: string;
+  */
+};
+
+export type ValidatorsAppValidator = {
+  activeStake: number;
+  commission: number;
+  totalScore: number;
+  voteAccount: string;
+  name?: string;
+  avatarUrl?: string;
+  wwwUrl?: string;
+};
+
+const URLS = {
+  validatorList: (cluster: Extract<Cluster, "mainnet-beta" | "testnet">) => {
+    const clusterSlug = cluster === "mainnet-beta" ? "mainnet" : cluster;
+    return `https://www.validators.app/api/v1/validators/${clusterSlug}.json?order=score`;
```

Comment:
@konoart At the same time you change the value to `http://validators-solana.coin.ledger.com`, it would be useful for us to get that server's URL from an env var, similar to `API_SOLANA_PROXY` for the node. So we could fallback to the direct endpoint for testing, especially while the new endpoint finishes being set up. Thanks. :slightly_smiling_face: